.github/workflows/auto-cherry-picker.yaml

```diff
@@ -23,6 +23,9 @@ jobs:
         with:
           # Always checkout the latest commit of main
           ref: "main"
+          # As an attempt to solve spurrious merge-conflicts, we'll fetch full history
+          fetch-depth: 0
```

Comment:
with 13k+ commits, couldn't an arbitrary large, but still significantly smaller number be used as "good enough", say fetch the last 2000 commits? (thats roughly ~100 days of development at current pace)

Reply:
I think it makes sense to try "get everything" as a first step, to validate if it's shallow clones causing problems. We can then try to optimise separately, once we've got it working properly.